curriculum/challenges/english/01-responsive-web-design/css-variables-skyline/part-003.md

```diff
@@ -7,18 +7,50 @@ dashedName: part-3
 
 # --description--
 
-Next, add opening and closing `head` and `body` tags within the html element.
+Next, add opening and closing `head` and `body` tags within the `html` element.
```

Comment:
I will add a test for that 👍

Reply:
Actually, the browser corrects this.
My general stance has been to allow things to pass the tests, if the browser knows what the Camper meant. Do you think we should write RegEx for this?

Reply:
I was just going through this PR and came across this. It's a tough call. Generally, I have been in favor of letting the tests pass if the browser renders it correctly. I think it's a minor problem (or maybe not a problem), in this case, and I would be fine leaving it as-is for now.

Reply:
Side note: The browser corrects this too...
```
</body>
</head>
</html>
<html>
<head>
<body>
```
Wow

Reply:
@moT01 Are you saying, in this challenge, you would prefer a test to be written?

Reply:
No, I don't think this is that big of an issue - and possibly better without a test. Either way works for me. I would say to leave it since it's already like this.